cfgov/preprocessed/js/molecules/Expandable.js

```diff
@@ -0,0 +1,251 @@
+'use strict';
+
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+require( '../modules/polyfill/event-listener' );
+require( '../modules/polyfill/class-list' );
+
+// Required modules.
+var EventObserver = require( '../modules/util/EventObserver' );
+var $ = require( 'jquery' );
+
+/**
+ * Expandable
+ * @class
+ *
+ * @classdesc Initializes a new Expandable molecule.
+ *
+ * @param {Object} context
+ *   The DOM element within which to search for the molecule.
+ * @returns {Object} An Expandable instance.
+ */
+function Expandable( context ) { // eslint-disable-line max-statements, inline-comments, max-len
+
+  // TODO: Remove this when difference between atomic Expandable
+  //       and cf-expandables is resolved.
+  // ==========================================================================
+  // Unsets global expandables logic.
+  if ( $.fn.expandable ) {
+    var html = context.innerHTML;
+    context.innerHTML = html;
+
+    var $this = $( context );
+    var $cueOpen = $this.find( '.expandable_cue-open' ).not( $this.find( '.expandable .expandable_cue-open' ) );
+    var $cueClose = $this.find( '.expandable_cue-close' ).not( $this.find( '.expandable .expandable_cue-close' ) );
+    var $content = $this.find( '.expandable_content' ).not( $this.find( '.expandable .expandable_content' ) );
+
+    $cueOpen.attr( 'style', '' );
+    $cueClose.attr( 'style', '' );
+    $content.attr( 'style', '' );
+
+  }
+  // ==========================================================================
+
+
+  // The Expandable context will directly be the Expandable when used in an ExpandableGroup.
+  var _dom = context.classList.contains( 'expandable' ) ?
+             context : context.querySelector( '.expandable' );
+  var _target = _dom.querySelector( '.expandable_target' );
+  var _content = _dom.querySelector( '.expandable_content' );
+  var _link = _dom.querySelector( '.expandable_link' );
+  var _cueOpen = _link.querySelector( '.expandable_cue-open' );
+  var _cueClose = _link.querySelector( '.expandable_cue-close' );
+  var _isAnimating = false;
+  var _clickQueue = 0;
+  var _isExpanded;
+  var _that = this;
+  var _contentHeight = $( _content ).height();
```

Comment:
Yeah, this assumes once the expandable is rendered by the backend the content will not be changing.

I'm using jQuery just because it's still used for the animation and computed style needs a polyfill. I'm experimenting with using CSS for the animation so will remove this too if that works.